frontend/src/components/pages/ManageUsersPage.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from "react";
+import { useQuery } from "@apollo/client";
+import { Box, Button, Flex, HStack, Text } from "@chakra-ui/react";
+import { DownloadIcon, SearchIcon } from "@chakra-ui/icons";
+import {
+  AdminPage,
+  UserCardProps,
+  UsersByEmail,
+} from "../../types/AdminDashboardTypes";
+import UserCard from "../admin/UserCard";
+import SideBar from "../admin/SideBar";
+import SearchUsersBar from "../admin/SearchUsersBar";
+import { UserResponse } from "../../APIClients/types/UserClientTypes";
+import { USERS } from "../../APIClients/queries/UserQueries";
+
+const NoUserSelectedCard = (): React.ReactElement => {
+  return (
+    <Box
+      align="center"
+      py={20}
+      border="1px"
+      borderColor="background.lightgrey"
+      borderRadius="6px"
+    >
+      <SearchIcon h="100px" w="100px" color="text.grey" />
+      <Text variant="display-md">No User Selected</Text>
+      <Text variant="body">
+        Use the search bar above to view specific user details
+      </Text>
+    </Box>
+  );
+};
+
+const ManageUsersPage = (): React.ReactElement => {
+  const [
+    spotlightedUser,
+    setSpotlightedUser,
+  ] = useState<UserCardProps | null>();
+
+  // Never expose this
+  const [users, setUsers] = useState<UsersByEmail>({});
+
+  useQuery<{
+    users: Array<UserResponse>;
+  }>(USERS, {
+    fetchPolicy: "cache-and-network",
+    onCompleted: (data) => {
+      const userProfiles = data.users.reduce(
+        (map: UsersByEmail, user: UserResponse) => {
+          // eslint-disable-next-line no-param-reassign
```

Comment:
we've used this workaround before, couldn't quite find what the expectation is here

Reply:
Yep, definitely should just disable the linter here. Only other solutions I see are to use object spreading, but that would create a new object for every call of the function which isn't exactly performant.